File: benchmark/comparison/results/blind_py/07.py
def fib(n):
    if n <= 0:
        return 0
    if n == 1:
        return 1
    a, b = 0, 1
    for _ in range(2, n + 1):
        a, b = b, a + b
    return b

print(fib(0))
print(fib(1))
print(fib(2))
print(fib(5))
print(fib(10))

File: benchmark/comparison/results/blind_ts/07.ts
function fib(n: number): number {
  if (n === 0) return 0;
  if (n === 1) return 1;
  let a = 0, b = 1;
  for (let i = 2; i <= n; i++) {
    const tmp = a + b;
    a = b;
    b = tmp;
  }
  return b;
}

console.log(fib(0));
console.log(fib(1));
console.log(fib(2));
console.log(fib(5));
console.log(fib(10));
